src/index.ts

```diff
@@ -2109,39 +2119,23 @@ export class GossipSub extends EventEmitter<GossipsubEvents> implements Initiali
     // First we collect the peers above gossipThreshold that are not in the exclude set
     // and then randomly select from that set
     // We also exclude direct peers, as there is no reason to emit gossip to them
```

Comment:
move these comments to somewhere thats more relevant or delete if there's nowhere that makes sense